### ISSUE TITLE
Pagination - Ensure color is not the sole means of communicating information - (2036570531)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5591]
### Module: 
 22b Pagination 

### Violation: 
 Ensure color is not the sole means of communicating information 

![image](https://user-images.githubusercontent.com/5023024/202748898-539247d3-20f3-4825-a33e-e84c41695673.png)

### WCAG Reference: 
- [1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) 

### Instance ID: 
 2036570531 

### Severity: 
 10 

### Description: 
 [Issue]
There is content that uses color alone to visually indicate information. Examples include:
- Current selected state of pagination component. 

[User Impact]
Users with color blindness will have difficulty identifying the colors used to convey this information.

[Code Reference] 
 ```
<button class="page is-selected">55</button>
 ``` 

### Note: 
 [Suggestion]
Ensure color is not the sole means of communicating information. Color can be used as long as other visual indicators of the information are also present. Common additional visual indicators include underlining, bolding, additional on-screen text conveying the information, or change in shape or size.

[Compliant Code Example] 
 ```
N/A
 ```
 

### Media Type: 
 Color and Contrast 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Not a valid bug, context is provided with both color and an underlined line when selected. Closing.
